csclient/params/params.go

```diff
@@ -221,6 +221,20 @@ type PermRequest struct {
 	Write []string
 }
 
+// HomePage holds the request/response of an id/meta/home-page PUT or GET request.
+// See https://github.com/juju/charmstore/blob/v4/docs/API.md#put-idmeta
+// See https://github.com/juju/charmstore/blob/v4/docs/API.md#get-idmetahome-page
+type HomePage struct {
```

Comment:
Except this is the same struct today but I could use 2 differents structs, not really sure it adds something today.